src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/view-index-top.jelly

```diff
@@ -36,5 +36,9 @@ THE SOFTWARE.
       </div>
       <j:out value="${it.description!=null ? app.markupFormatter.translate(it.description) : ''}"/>
   </div>
+  <!-- give actions a chance to contribute summary item -->
+  <j:forEach var="a" items="${it.allActions}">
+    <st:include page="summary.jelly" from="${a}" optional="true" it="${a}" />
```

Comment:
:ant: better style is to say `page="summary"` since we do not presume the implementing Stapler facet. (The `.jelly` suffix will just be ignored in this case but it is misleading.)